lib/hybrid-sdk/common/config/config.ts

```diff
@@ -31,7 +31,12 @@ export const findProjectRoot = (startDir: string): string | null => {
     currentDir = path.dirname(currentDir);
   }
 
-  return null;
+  const errorMessage =
+    'Error: config.default.json is missing, please ensure the file exists when running the broker.';
+  logger.error(errorMessage);
```

Comment:
If we;re not handling the error, generally we shouldn;t log it, otherwise we;ll get an error logged at every layer of the stack until it is handled

Reply:
Fair enough, I removed the extra log message